### ISSUE TITLE
Exception on startup: BinderProxy cannot be cast to ActivityRecord$Token

### ISSUE THREAD

**Issue by cweiske** [tobexyz/yaacc-code#15]
I installed YAACC 2.1.1 freshly on a Nexus 7 with LineageOS (Android 7.1.2, Linage 14.1-20181227-flo).

On every start, I get the following exception 5 times in a row (I have 4 renderers and 1 server):
```
I ActivityManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=de.yaacc/.browser.TabBrowserActivity (has extras)} from uid 10026 on display 0
D ACDB-LOADER: ACDB -> send_afe_cal
D audio_hw_primary: enable_audio_route: usecase(1) apply and update mixer path: low-latency-playback
W ActivityManager: Bad activity token: android.os.BinderProxy@9225e9d
W ActivityManager: java.lang.ClassCastException: android.os.BinderProxy cannot be cast to com.android.server.am.ActivityRecord$Token
W ActivityManager: 	at com.android.server.am.ActivityRecord.forTokenLocked(ActivityRecord.java:594)
W ActivityManager: 	at com.android.server.am.ActivityRecord.isInStackLocked(ActivityRecord.java:1347)
W ActivityManager: 	at com.android.server.am.ActivityRecord.getStackLocked(ActivityRecord.java:1352)
W ActivityManager: 	at com.android.server.am.ActivityManagerService.getActivityDisplayId(ActivityManagerService.java:10061)
W ActivityManager: 	at android.app.ActivityManagerNative.onTransact(ActivityManagerNative.java:2567)
W ActivityManager: 	at com.android.server.am.ActivityManagerService.onTransact(ActivityManagerService.java:2974)
W ActivityManager: 	at android.os.Binder.execTransact(Binder.java:565)
```

Other apps do not cause this exception on startup.
YAACC still works, despite this exception.

This exception also repeats 5 times when I click the "receivers" tab the first time, and when I click the "player" tab the first time.

**Comment by cweiske** [comment]
This exception actually has nothing to do with YAACC. It simply hangs without any exception at all.

**Comment by cweiske** [comment]
Also on 2.2.0

**Comment by tobexyz** [comment]
fixed with version 2.3.0

**Comment by cweiske** [comment]
Still happens with 2.3.0 in the debug build.
